block_test.go

```diff
@@ -1601,7 +1601,9 @@ func TestTOC(t *testing.T) {
 		"#",
 		"",
 	}
-	doTestsBlock(t, tests, TOC)
+	doTestsParam(t, tests, TestParams{
+		HTMLFlags: UseXHTML | TOC,
```

Comment:
It's a bit obscure: `doTestsBlock` passes `UseXHTML` down to `doTestsParam` and since I'm switching to this "lower level" func, the flag bubbled up.